src/extension.ts

```diff
@@ -70,3 +75,16 @@ async function withSpinningStatus(context: vscode.ExtensionContext, action: (sta
 
 // this method is called when your extension is deactivated
 export function deactivate(): void {}
+
+class ExtensionAPI {
+
+    private kotlinApi?: KotlinAPI;
+
+    constructor(kotlinApi: KotlinAPI) {
+        this.kotlinApi = kotlinApi;
+    }
+
+    getBuildOutputPath(): string {
```

Comment:
Can we use ES6-style getters, i.e., something along the lines of

```suggestion
    get buildOutputPath(): string {
```

?